Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <VlingoVersion>0.6.4.6</VlingoVersion>
+  </PropertyGroup>
+</Project>
```

Comment:
@Johnny-Bee Just for me to understand. If I'm only working on `vlingo-net-uuid` repo. How the version from this file will be injected to `Vlingo.UUID.Tests.csproj` ? I cannot see it is included

Reply:
The test projects are using project reference so no worries

Reply:
The repos are still independent

Reply:
Yes the usage of project references is triggered when you use the solution configuration Debug with Project References

Reply:
Seems like this is imported directly during the build according to https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019